tests/spyre_util.py

```diff
@@ -528,22 +566,42 @@ def create_text_prompt(model: str, min_tokens: int, max_tokens: int) -> str:
     return prompt
 
 
-def create_random_request(
-        request_id: int, num_tokens: int,
-        sampling_params: SamplingParams) -> EngineCoreRequest:
+def create_random_request(request_id: int,
+                          num_tokens: int,
+                          sampling_params: SamplingParams,
+                          from_model_vocab: bool = False,
```

Comment:
It looks like this is never set to true now, so this method no longer creates any random requests

Reply:
I wonder if we still need `set_random_seed` now though 🤔